Tasks/Xcode/xcode.ts

```diff
@@ -294,7 +294,7 @@ async function run() {
                 let exportTeamId: string;
                 let exportOptionsPlist: string;
                 let archiveToCheck: string = archiveFolders[0];
-                let embeddedProvProfiles: string[] = tl.findMatch(archiveToCheck, '**/embedded.mobileprovision', { followSymbolicLinks: false, followSpecifiedSymbolicLink: false });
+                let embeddedProvProfiles: string[] = tl.findMatch(archiveToCheck, '**/embedded.*provision*', { followSymbolicLinks: false, followSpecifiedSymbolicLink: false });
```

Comment:
I don't know if there could be other types here but it feels safer to check for **/embedded.mobileprovision (iOS) and if nothing is found, then check for **/embedded.provisioningprofile (macOS) explicitly.

Reply:
I'll change this to check twice. But I intend to assign the result to embeddedProvProfiles either way, as both iOS and macOS provisioning profiles can have iCloud entitlements so I think we want to call with embeddedProvProfiles[0].

Reply:
In a single archive, do you know if there can be both .mobileprovision and .provisioningprofile?

Reply:
I don't think so. I tried creating a Cross Platform game, and each platform got its own scheme:

```
$ xcodebuild -workspace Xcode9CrossPlatformGame.xcodeproj/project.xcworkspace -list
Information about workspace "Xcode9CrossPlatformGame":
    Schemes:
        Xcode9CrossPlatformGame iOS
        Xcode9CrossPlatformGame tvOS
        Xcode9CrossPlatformGame watchOS App
        Xcode9CrossPlatformGame macOS
```

The scheme argument is required for the archive action, so this would produce 4 archives.

Reply:
Thanks!